### DISCUSSION TITLE
[Proposition] Ajouter l'action personnalisée DUPLICATE pour gérer les Pull request dupliquées

### DISCUSSION THREAD

**Discussion by Mozinou2013** [INFOM126-Automated-Software-Engineering/2048-groupe-07#25] [New GitHub action]
Nom de l'action proposée : DUPLICATE
Objectif de l'action (décrire ce qu'elle fait et pourquoi elle est nécessaire) : Gérer les pull requests qui sont dupliquées
Fonctionnement (expliquer brièvement les étapes et les technologies utilisées) : La github action remarquerait qu'une pull request est dupliquée et comme ça on l'annulerait

**Comment by ldelpier** [comment]
Quels fichiers seraient ajoutés au projet

**Reply by Mozinou2013** [reply]
Un fichier yml dans les workflows et une documentation si celle-ci peut être utile 

Qu'en pensez-vous dans ce cas ?

**Reply by ldelpier** [reply]
Très bonne idée ! Faites donc ;)

**Reply by Mozinou2013** [reply]
Merci de votre soutien !
